src/main/java/com/parth/importer/repository/LogRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.parth.importer.repository;
+
+import com.parth.importer.model.Log;
+import org.apache.ibatis.annotations.Mapper;
+
+@Mapper
+public interface LogRepository {
+    void addLogs(Log log);
```

Comment:
```suggestion
    void addLog(Log log);
```
Since we are adding a single log

Reply:
Done